src/isyntax/isyntax.c

```diff
@@ -1416,10 +1478,9 @@ static void convert_ycocg_to_bgra_block(icoeff_t* Y, icoeff_t* Co, icoeff_t* Cg,
 }
 
 static void convert_ycocg_to_rgba_block(icoeff_t* Y, icoeff_t* Co, icoeff_t* Cg, i32 width, i32 height, i32 stride, u32* out_rgba) {
-    i32 first_valid_pixel = ISYNTAX_IDWT_FIRST_VALID_PIXEL;
     i32 aligned_width = (width / 8) * 8;
 
-    for (i32 y = 0; y < aligned_width; ++y) {
```

Comment:
I guess it was working for square tiles because aligned_width == height? Just curious, no action needed.

Reply:
Yes, I guess so! I happened to notice the mistake.